fennel/expr/test_expr.py

```diff
@@ -0,0 +1,316 @@
+import pytest
+import pandas as pd
+from datetime import datetime
+from typing import Dict
+from fennel.datasets import dataset
+
+from fennel.expr import F, when
+from fennel.expr.visitor import ExprPrinter
+from fennel.expr.serializer import ExprSerializer
+from google.protobuf.json_format import ParseDict  # type: ignore
+from fennel.gen.expr_pb2 import Expr
+from fennel.testing.test_utils import error_message
+from fennel.expr.utils import compute_expr, compile
```

Comment:
rename compute_expr -> `eval` and compile -> `typeof`

Reply:
You could also consider adding them as methods on `Expr` itself instead of adding them to utils.